site/content/en/docs/Contributing/releasing.en.md

```diff
@@ -110,7 +110,7 @@ Verify release checksums by running `make check-release`
 
 ## Update docs
 
-If there are major changes, please send a PR to update <https://kubernetes.io/docs/setup/minikube/>
```

Comment:
This link also pointing to the same page. no 404 here

Reply:
I understand what you mean, I just want to update it.